### ISSUE TITLE
Render Issue With Hardcore Darkness

### ISSUE THREAD

**Issue by givemhell** [CptSpaceToaster/CptsModdingLight#49]
CptSpaceToaster it seems i found another mod that makes things act funny 

http://www.curse.com/mc-mods/minecraft/225957-hardcore-darkness
"it removes the minimum block & sky light so if there is no light source its actually dark"
"I really am just editing the vanilla lightmap to not have that minimal light."

this is the result
http://i.imgur.com/V9So2rT.png <-- night
http://i.imgur.com/MEKH6LW.png <-- day

i also let the other mod author know of the problem, but hes not all to sure on how to fix it
https://github.com/lumien231/Hardcore-Darkness/issues/5

**Comment by CptSpaceToaster** [comment]
Yeah, I basically removed the entire lightmap texture system (it's still there, just unused)

Hardcore darkness will fire off it's transformers, but it won't change any behavior.  I'll have to check to see if hardcore darkness exists, and then implement it myself basically....

Or just get around to creating that configuration file I haven't made yet.

**Comment by asiekierka** [comment]
Seconding.

**Comment by CptSpaceToaster** [comment]
:+1:

**Comment by givemhell** [comment]
we wont see a fix for 1.7 right?
